doc/requirements/requirements.md

```diff
@@ -1002,17 +1002,33 @@ PF **MAY** support at least:
 needs emerge to use it on other platform.</why>
 
 ## Tuning
-### Get and set multiple parameter values in one request.
-Get and set the content of one or more parameter or multiple parameters
-as xml.
+### Get and set multiple parameter values in one request
+#### Consistency
+When setting multiple parameters from one client request,
+and when one ore more parameter value is invalid (eg. out of range),
+no parameter **SHOULD** be set.
+Eg: an invalid request to change parameters **SHOULD** not impact the parameters
+values nor the subsystems.
+<note>This can be implemented by first checking parameters validity
+before setting them, or implementing a rollback mechanism.</note>
+<why>To provide a parameter database consistency to the client.
+This is especially important in the case of exclusive parameter values.
+Eg: let two parameters have excluding values,
+if a transaction fail after the first parameter is set but not the second,
+the excluding constraint may be violated.</why>
+
+#### Access parameters as Xml
+Getting and setting the content of one or more parameters **SHOULD** be possible in xml.
 <why>For performance reason. Tools often need to update multiple parameter
-and having one call per parameter is too slow. (benchmark ?)</why>
+and having one call per parameter is too slow. (benchmark ?).
+This feature permit the client to save and restore from an external database parameter
+values a la `alsa.state`.</why>
```

Comment:
:+1: I double checked it is called (asound.state)[https://wiki.archlinux.org/index.php/Advanced_Linux_Sound_Architecture#ALSA_and_Systemd]. `alsa-restore.service` and `alsa-store.service` are the functions/service that restore and save the state.